system/jlib/jdebug.cpp

```diff
@@ -1797,8 +1797,8 @@ class CExtendedStats  // Disk network and cpu stats
         while (fgets(ln, sizeof(ln), netfp)) {
             const char *s = ln;
             skipSp(s);
-            size_t ilen = strlen(ifname);
-            if ( (strncmp(s, ifname, ilen)==0) && ((ilen<sizeof(ln)) && (s[ilen]==':')) ) {
+            size_t ilen = ifname.length();
+            if ( (strncmp(s, ifname.str(), ilen)==0) && ((ilen<sizeof(ln)) && (s[ilen]==':')) ) {
                 s+=ilen;
```

Comment:
s += ilen+1
I think.

Reply:
fgets() always null terminates, so no need for the ilen<sizeof() test since always true.  (There was no equivalent in the previous code.)

Reply:
fixed